docs/documentation/platform/identities/oidc-auth/gitlab.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: GitLab
+description: "Learn how to authenticate GitLab pipelines with Infisical using OpenID Connect (OIDC)."
+---
+
+**OIDC Auth** is a platform-agnostic JWT-based authentication method that can be used to authenticate from any platform or environment using an identity provider with OpenID Connect.
+
+## Diagram
+
+The following sequence diagram illustrates the OIDC Auth workflow for authenticating GitLab pipelines with Infisical.
+
+```mermaid
+sequenceDiagram
+  participant Client as GitLab Pipeline
+  participant Idp as Identity Provider
```

Comment:
I see in the CircleCI OIDC docs it says "CircleCI Identity Provider." I'm thinking we should stick to this and say here either "GitLab," "Gitlab IdP," or "GitLab Identity Provider" so there is no ambiguity over what Identity Provider is referring to here.

What do you think? If so, feel free to adjust the others (GitHub, CircleCI) as well to be consistent.